avocado/core/tree.py

```diff
@@ -510,7 +505,7 @@ def process_node(node):
         Generate this node's tree-view
         :return: list of lines
         """
-        if node.multiplex:
+        if getattr(node, "multiplex", None):
```

Comment:
I'd prefer keeping it this way for now, because there are already so many changes. When we have this part working we can address the tree-view (as I mentioned somewhere I'd prefer having `tree_view` method inside the tree-class, but let's explore the possibilities after those changes).